### ISSUE TITLE
Adapt code to monetization

### ISSUE THREAD

**Issue by ctchervenkov** [matsim-eth/SCCER-Pipeline#21]
Current, externalities (emissions, congestion) are calculated for each GPS trace.
To be included is the calculation of costs directly in the loop.
Some refactoring will be needed.

**Comment by joemolloy** [comment]
for the monetisation of emssisons, here is a list of the different types we can collect:
[http://www.hbefa.net/e/help/HBEFA33_help_en.pdf] (page 21)

currently, the code only collects the following:
CO("CO"), CO2_TOTAL("CO2(total)"), FC("FC"), HC("HC"), NMHC("NMHC"), NOX("NOx"), NO2("NO2"), PM("PM"),	SO2("SO2");

**Comment by joemolloy** [comment]
But then, to add other emissions, ill have to re-export the hbefa database, and then edit the emissions contribute to support them. Not too much work

**Comment by ctchervenkov** [comment]
That would be great to basically include all the pollutants that are available in HBEFA.
I would also not mind to do it if you quickly show me where the database lives.

**Comment by ctchervenkov** [comment]
Yeah, for methane CH4 being part of HC, we still need to know what percentage of this total HC is CH4, no?
